### ISSUE TITLE
2.0.2 npm ERR! cb() never called!

### ISSUE THREAD

**Issue by adinvadim** [BenoitZugmeyer/eslint-plugin-html#53]
on yarn install
```
error An unexpected error occurred: "https://registry.yarnpkg.com/eslint-plugin-html/-/eslint-plugin-html-2.0.2.tgz: invalid tar file".
```

on npm install eslint-plugin-html@2.0.2
```
npm ERR! cb() never called!

npm ERR! This is an error with npm itself. Please report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
```

**Comment by karamanolev** [comment]
Related bug report from npm with more debugging information: https://github.com/npm/npm/issues/16551

**Comment by bleepsandblops** [comment]
Same issue here. Worked around it by install previous version 
`npm install eslint-plugin-html@2.0.1`

**Comment by BenoitZugmeyer** [comment]
Thank you for reporting, I'll investigate.

**Comment by nkovacs** [comment]
It might just be a corrupted upload: https://github.com/binarykitchen/spider-detector/issues/7#issuecomment-295927780

**Comment by yoyo837** [comment]
Same issue.

Invalid tar header. Maybe the tar is corrupted or it needs to be gunzipped?

**Comment by BenoitZugmeyer** [comment]
I just republished my package, it should work now.

**Comment by nkovacs** [comment]
Thanks, it works now.

**Comment by BenoitZugmeyer** [comment]
I don't know why, but `yarn publish` fails or produce invalid packages. Using `npm publish` works though.

**Comment by nkovacs** [comment]
Yarn bug: https://github.com/yarnpkg/yarn/issues/2764

**Comment by BenoitZugmeyer** [comment]
Ha, thank you very much I was looking for this :)
